### PR TITLE
Persistent mode always

### PR DESCRIPTION
**Pull request by domenukk**
This PR always enables persistent mode, as discussed in #1565

### CODE REVIEW

**Comment by domenukk** [comment]
The logic is simple: if it's running inside afl, it's persistent (loop count >1), else it's not and runs but once.

File: instrumentation/afl-compiler-rt.o.c [hunk 1]
```diff
@@ -105,6 +105,9 @@ u32 __afl_dictionary_len;
 u64 __afl_map_addr;
 u32 __afl_first_final_loc;
 
+/* 1 if we are running in afl, and the forkserver was stared, else 0 */
```
Comment (vanhauser-thc) [inline instrumentation/afl-compiler-rt.o.c:108]: typo

**Comment by vanhauser-thc** [comment]
the persistent signature must still be preserved as otherwise this will break oss-fuzz in this coming PR https://github.com/google/oss-fuzz/pull/7343/files

**Comment by domenukk** [comment]
It is preserved

**Comment by vanhauser-thc** [comment]
can you please fix the typo?
also, what is the benefit this creates? as it is this this is just a different way and does not improve anything, or am I missing something?

**Comment by domenukk** [comment]
It'll enable persistent mode in LibAFL for relevant targets, without the need to scan the binary or set an env

**Pull request merged**